LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua

```diff
@@ -121,10 +185,23 @@ local function CreateLabelAndContainerControl(parent, controlData, controlName)
         label:SetAnchor(TOPRIGHT, container, TOPLEFT, 5, 0)
     end
 
+    local anchorFAQX = 0
+    local anchorFAQPos = LEFT
+    local anchorFAQRelativePos = LEFT
+    if label.GetTextWidth then
+        anchorFAQX = label:GetTextWidth() + 5
+    else
+        anchorFAQX = -30
+        anchorFAQPos = LEFT
+        anchorFAQRelativePos = RIGHT
+    end
+    CreateFAQTexture(control, controlData, controlName, anchorFAQPos, label, anchorFAQRelativePos, anchorFAQX, 0)
+
     control.data.tooltipText = GetStringFromValue(control.data.tooltip)
     control:SetMouseEnabled(true)
-    control:SetHandler("OnMouseEnter", ZO_Options_OnMouseEnter)
-    control:SetHandler("OnMouseExit", ZO_Options_OnMouseExit)
+    control:SetHandler("OnMouseEnter", ZO_Options_OnMouseEnter, "LAM2_Control_OnMouseEnter")
```

Comment:
Is this change needed? If possible I'd like to avoid changing the current behaviour in unexpected ways. If anyone tries to hook the OnMouseEnter handler and expects to be able to prevent the LAM handler from running, it would break due to this.

Reply:
Not sure why this should break? It's ZOs code and if this fails to load properly because there are PreHooks or PostHooks they should secure it or the devs need to adopt their code either way, I'd say.
Or I don't understand what the problem would be.

I had added this as I was using the OnMouseEnter -> Show/hide the faq texture. As you already wished the textrues to always show I can remove it again.